app/views/patients/show.html.erb

```diff
@@ -126,10 +130,14 @@
           <td><%= cc.last_name %></td>
           <td><%= cc.primary_telephone.present? && cc.primary_telephone.length > 10 ? cc.primary_telephone&.gsub('+1', '')&.gsub(/\D/, '')&.insert(3, '-')&.insert(7, '-') : '' %></td>
           <td><%= cc.email %></td>
-          <td><%= cc.contact_attempts %></td>
+          <td class="text-center"><%= cc.last_date_of_exposure&.to_date&.strftime('%m/%d/%Y') || 'Unknown' %></td>
```

Comment:
Did the PO team request that it say "Unknown" here? I think typically we leave it blank in scenarios like this.

Reply:
After some discussion with the PO team, this one is still up in the air. Going to leave it blank, but that may change based on feedback. People seem split about this one.